tst/SdpApiTest.cpp

```diff
@@ -1702,6 +1712,7 @@ TEST_P(SdpApiTest_SdpMatch, populateSingleMediaSection_TestH264Fmtp)
     MEMSET(&rtcConfiguration, 0x00, SIZEOF(RtcConfiguration));
     MEMSET(&track1, 0x00, SIZEOF(RtcMediaStreamTrack));
     MEMSET(&rtcSessionDescriptionInit, 0x00, SIZEOF(RtcSessionDescriptionInit));
+    MEMSET(&rtcRtpTransceiverInit, 0x00, SIZEOF(RtcRtpTransceiverInit));
```

Comment:
One concern I have is that not all applications zero this input parameter (transceiver settings), would suggest looking into a way that doesn't require this

Reply:
```
pKvsRtpTransceiver = (PKvsRtpTransceiver) MEMCALLOC(1, SIZEOF(KvsRtpTransceiver));
```
It actually gets zeroed internally in the SDK in `createKvsRtpTransceiver()` [here](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/30f52d18e647747de1ec29ad6f6d3f62128f5494/src/source/PeerConnection/Rtp.c#L16) which gets called when the application calls `addTransceiver()`.

We only it on the application-side for the tests and not in the samples apps.